alacritty_terminal/tests/ref.rs

```diff
@@ -48,6 +48,7 @@ ref_tests! {
     vttest_origin_mode_2
     vttest_scroll
     vttest_tab_clear_set
+    vttest_selective_erasure
```

Comment:
The `vttest` are named this way because they were created with the `vttest` application. Looking at your test it doesn't look like that's the case. So you should probably rename it to prevent confusion, simply dropping the `vttest_` should do the job.